hs_core/views/resource_folder_hierarchy.py

```diff
@@ -318,7 +318,8 @@ def zip_aggregation_file(request, res_id=None):
         return JsonResponse({"error": "Permission denied"}, status=status.HTTP_401_UNAUTHORIZED)
 
     if resource.resource_type != "CompositeResource":
-        return JsonResponse({"error": "Resource is not of type composite resource"}, status=status.HTTP_400_BAD_REQUEST)
+        err_msg = f"{resource.display_name} type doesn't support zipping of aggregation."
```

Comment:
Unzipping within the file browser failed for me locally, within a standard Composite resource. Not sure if this was something about my build, so worth testing on your end.
![image](https://user-images.githubusercontent.com/17934193/207618112-f0143369-b3be-48f1-9bf4-0ed4b4d59d00.png)